core/src/util/vboMesh.cpp

```diff
@@ -73,10 +71,10 @@ void VboMesh::subDataUpload() {
     if (m_dirtySize != 0) {
         glBindBuffer(GL_ARRAY_BUFFER, m_glVertexBuffer);
 
-        long vertexBytes = m_nVertices * m_vertexLayout->getStride();
+        size_t vertexBytes = m_nVertices * m_vertexLayout->getStride();
 
         // updating the entire buffer
-        if (std::abs(m_dirtySize - vertexBytes) < m_vertexLayout->getStride()) {
+        if (vertexBytes - m_dirtySize < m_vertexLayout->getStride()) {
 
```

Comment:
Anyone knows where this change comes from? -  this is not necessarily part of header-cleaning.